Source/Core/DiscIO/Enums.cpp

```diff
@@ -182,6 +182,8 @@ std::string GetSysMenuVersionString(u16 title_version)
   case Region::NTSC_K:
     region_letter = "K";
     break;
+  case Region::UNKNOWN_REGION:
```

Comment:
I don't see much point in logging, but... eh, whatever.

Showing a system menu version without a region letter isn't that strange. It's not uncommon to just say "4.3" when talking about 4.3 even if the specific version you're running actually is 4.3E or 4.3U or whatever. On the other hand, if we put a question mark there, it's not obvious that it means that the region is unknown. If you would see Dolphin say "Launch Wii System Menu 3.0?", don't you think it looks more like the sentence is a question?